addons/point_of_sale/static/src/js/models.js

```diff
@@ -175,7 +175,7 @@ exports.PosModel = Backbone.Model.extend({
         loaded: function(self,users){
             self.user = users[0];
             self.user.role = 'manager';
-            self.employee.name = self.user.name;
+            self.employee = self.user;
```

Comment:
Why do you change it here? If really needed maybe do a separate commit

Reply:
Probably a leftover from before spec change, like the always true argument earlier. Removing it :wink: